xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -114,9 +141,17 @@
         <ReturnType>System.Text.Json.JsonNamingPolicy</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the policy used to convert a <see cref="T:System.Collections.IDictionary" /> key's name to another format, such as camel-casing.</summary>
+        <value>The policy used to convert a <see cref="T:System.Collections.IDictionary" /> key's name to another format.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This property can be set to <xref:System.Text.Json.JsonNamingPolicy.CamelCase?displayProperty=nameWithType> to specify a camel-casing policy.
+
```

Comment:
Add the remark, "It is not used when deserializing." Why didn't the tool pick that up?
https://github.com/dotnet/corefx/blob/cc5653fcceefce5eac38dc7847b12b371027f78f/src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs#L117-L121

It was modified recently (~3 weeks ago), and maybe what you were diffing against was out-dated.

Reply:
```suggestion

This property can be set to <xref:JsonNamingPolicy.CamelCase> to specify a camel-casing policy.

It is not used when deserializing.

```